### PR TITLE
Dev doha

### PR DESCRIPTION
**Pull request by dohmeid**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    </main>
+
+
+    <footer>
+        <small>Copyright &copy; 2024 Doha Hmeid. All Rights Reserved</small>
+    </footer>
+
+</body>
+
+</html>
```
Comment (wafa200099) [inline index.html:384]: I prefer to  add more details to your redme file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+        </section>
+
+
+        <!-- projects section -->
+        <section class="projects-page" id="projects-page">
+            <h2> My Projects</h2>
+            <ul>
+                <li>
+                    <div class="img-container">
+                        <img src="assets/images/projects/secret-code-game.png">
```
Comment (wafa200099) [inline index.html]: img should have **alt** attribute please add it to all images

Comment (dohmeid) [reply index.html]: Oh yeah, I forgot that! Thanks, I'll add it

File: index.html (hunk 1 above)
Comment (dohmeid) [reply index.html:384]: You're right, I was going to add details after your review and fixing the final edits

**Pull request merged**